gustaf/io/mixd.py

```diff
@@ -179,26 +179,12 @@ def export(
 
     # write connec
     with open(connec_file, "wb") as cf:
-        for c in (mesh.elements().ravel() + 1):
+        for c in (mesh.elements.ravel() + 1):
```

Comment:
This is I think already part of the splinepy pr. I stumbled over this error as well.... There are more like this in the file.

Reply:
Okay, yes I thought I got them all ;)